### ISSUE TITLE
Fix capitalization/naming of parameters and order of processing parameters in dropdowns

### ISSUE THREAD

**Issue by adswafford** [biocore/qiita#2236]
The dropdown text and processing parameters naming/capitalization should be improved as shown below. In addition, text in dropdown menus should be alphabetical.

![image](https://user-images.githubusercontent.com/27017606/29294436-2f8587fa-8104-11e7-8f36-f3db5d52f42c.png)

Multiplexed FASTQ; generic 5 base pair barcodes
Multiplexed FASTQ; generic 5 base pair barcodes with Phred quality threshold: 0
Multiplexed FASTQ; generic 5 base pair reverse complement mapping file barcodes
Multiplexed FASTQ; generic 6 base pair barcodes
Multiplexed FASTQ; generic 6 base pair reverse complement mapping file barcodes
Multiplexed FASTQ; generic 8 base pair barcodes
Multiplexed FASTQ; generic 8 base pair barcodes with Phred offset: 33
Multiplexed FASTQ; generic 8 base pair reverse complement mapping file barcodes
Multiplexed FASTQ; generic 11 base pair barcodes
Multiplexed FASTQ; generic 11 base pair reverse complement barcodes
Multiplexed FASTQ; generic 12 base pair barcodes
Multiplexed FASTQ; generic 12 base pair reverse complement barcodes
Multiplexed FASTQ; Golay 12 base pair barcodes
Multiplexed FASTQ; Golay 12 base pair barcodes with Phred offset: 33
Multiplexed FASTQ; Golay 12 base pair barcodes with Phred offset: 64
Multiplexed FASTQ; Golay 12 base pair reverse complement  barcodes
Multiplexed FASTQ; Golay 12 base pair reverse complement barcodes with Phred offset: 33
Multiplexed FASTQ; Golay 12 base pair reverse complement barcodes with Phred offset: 64
Multiplexed FASTQ; Golay 12 base pair reverse complement mapping file barcodes with reverse complement barcodes
Per-sample FASTQ defaults (autodetect)
Per-sample FASTQs; Phred offset: 33 (Sanger, Illumina 1.8+)
Per-sample FASTQs; Phred offset: 64 (Solexa, Illumina 1.3-1.7)

![image](https://user-images.githubusercontent.com/27017606/29296449-459e3716-810f-11e7-9656-a9ab64fc79ea.png)

Deblur (trimmed data, =<150 bp)
Pick closed reference OTUs [<- this is already correct]
Trim sequences (choose length below)

![image](https://user-images.githubusercontent.com/27017606/29296517-a83e1918-810f-11e7-9bfe-c487329213fd.png)

90 base pairs
100 base pairs (recommended for 16S)
150 base pairs
300 base pairs

![image](https://user-images.githubusercontent.com/27017606/29296629-649f7548-8110-11e7-8e74-76403c5f5e76.png)

Jobs to start
Positive reference filtering database
Threads per sample
Insertion/deletion (indel) probability 
Negative (artifacts) filtering database
Mean per nucleotide error rate
Error probabilities for each Hamming distance
Indexed version of the negative filtering database
Sequence trim length (-1 for no trimming)
An indexed version of the positive filtering database
Maximum number of allowed indels
Minimum study-wide read threshold
Minimum per-sample read threshold

**Comment by josenavas** [comment]
Thanks @adswafford 

I've updated the Default Parameter set names as you suggested here. The only difference is this one: `100 base pairs (recommended for 16S)` which I've just added `100 base pairs`. The recommendation is incorrect, and it is dependent on your technology. E.g. all our recent sequencing runs are 150bp, so there is no reason to trim to 100 unless we want to do a meta-analysis with a study that has shorter reads. In general, the longer the reads the better.

The other recommendations stated in this issue (Command name changes AND specific parameter names) are not changed because those require changes on the plugins. I've moved the issue to the current development cycle.
